components/scripts/lib/summary.sh

```diff
@@ -270,7 +270,7 @@ print_executed_non_cacheable_tasks() {
     printf -v value "%$1s %ss, %s total execution time" \
       "${executed_not_cacheable_num_tasks[1]}" \
       "${BUILD_TOOL_TASK}" \
-      "$(format_duration executed_not_cacheable_duration[1])"
+      "${executed_not_cacheable_duration[1]}"
   fi
```

Comment:
Where in the java client are we now formatting where we did not format before? I was not able to see in the javacode we now added any new formatting that was not there already before.

Reply:
Here:

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/dd2faf3aabc00d7d0c7aa635fa891664bfb77c9f/components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildTimeMetricsFields.java#L13-L17

And here:

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/dd2faf3aabc00d7d0c7aa635fa891664bfb77c9f/components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildScanDataFields.java#L80

Reply:
I don't understand the first link. There, formatting in the client was already done on the java client side before this PR.

Reply:
Got it. All clear now.